beta/src/pages/apis/createRoot.md

```diff
@@ -194,9 +235,13 @@ const domNode = document.getElementById('root');
 const root = createRoot(domNode);
 ```
 
-React will create a root for the `domNode`, and take over managing the DOM inside it.
+React will create a root for the `domNode`, and take over managing the DOM inside it. After you've created a root, you need to call [`root.render`](#root-render) to display a React component inside of it:
 
-An app fully built with React will usually only have one `createRoot` call with its root component.  A page that uses "sprinkles" of React for parts of the page may have as many `render` calls as needed.
+```js
+root.render(<App />);
+```
+
+An app fully built with React will usually only have one `createRoot` call for its root component. A page that uses "sprinkles" of React for parts of the page may have as many separate roots as needed.
```

Comment:
I tried to remove "sprinkles" here because it's not clear what that actually means.

Reply:
ok feel free to rephase